src/sentry/notifications/notifications/activity/base.py

```diff
@@ -88,7 +88,10 @@ def get_group_link(self) -> str:
         # method only used for emails
         # TODO: pass in recipient so we can add that to the referrer
         referrer = self.get_referrer(ExternalProviders.EMAIL)
-        return str(self.group.get_absolute_url(params={"referrer": referrer}))
+        params = {"referrer": referrer}
+        if hasattr(self, "notification_uuid") and self.notification_uuid:
```

Comment:
@jangjodi when would we not have a `notification_uuid`?

Reply:
you're right, this would never happen. I've changed it to reflect that